src/components/AssetLogo.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { TokenInfo } from '@alephium/token-list'
+import styled, { css } from 'styled-components'
+
+import AlephiumLogoSVG from '@/images/alephium_logo_monochrome.svg'
+
+interface AssetLogoProps {
+  asset: Pick<TokenInfo, 'id' | 'logoURI'>
+  size: number
+  className?: string
+}
+
+const AssetLogo = ({ asset, className }: AssetLogoProps) => (
+  <div className={className}>
+    <LogoImage src={asset.logoURI ?? AlephiumLogoSVG} />
+  </div>
+)
+
+export default styled(AssetLogo)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: ${({ size }) => size}px;
+  height: ${({ size }) => size}px;
+  border-radius: ${({ size }) => size}px;
+  padding: 5px;
+  flex-shrink: 0;
+
+  ${({ asset, theme }) =>
+    asset.id === '0' // ALPH
```

Comment:
In the desktop wallet, I created an object to identify the ALPH asset: https://github.com/alephium/desktop-wallet/blob/v2.0/src/storage/app-state/slices/assetsInfoSlice.ts#L28-L33

@h0ngcha0 came to me last week and asked me whether we should introduce such an object in the @alephium/token-list package and I think it's a great idea.

@h0ngcha0, did you move ahead with this, or should I take care of that?

@mvaivre do you agree?

EDIT: Just saw that you ported it from the desktop wallet later in this diff 😅

Reply:
Yeah, I thought about this as well! I agree, this should be in the token-list package :)